node/src/service.rs

```diff
@@ -182,7 +208,14 @@ where
 
     let validator = parachain_config.role.is_authority();
     let prometheus_registry = parachain_config.prometheus_registry().cloned();
-    let transaction_pool = params.transaction_pool.clone();
+
+    // Create filtered pool for RPC and offchain workers  
+    let filter = create_extrinsic_filter(
```

Comment:
If a client that is consuming the AvN pallets and service code wanted to customise the list of filtered extrinsics, how would they do it? Do they have to own the node and customise the service?